react-native-libraries.json

```
@@ -12229,5 +12229,15 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/hsjoberg/react-native-turbo-sqlite",
+    "examples": [
+      "https://github.com/hsjoberg/react-native-turbo-sqlite#usage"
```

Thanks for adding new library @hsjoberg! 🎉 

```suggestion
      "https://github.com/hsjoberg/react-native-turbo-sqlite/tree/master/example"
```

Just a nit, complete examples are preferred over in-docs examples. Otherwise looks good!

---

@Simek Ah gotcha. Thanks for informing me.